styles.css

```diff
@@ -506,3 +528,406 @@ input[type="email"]:focus {
   border-width: 5px;
   border-radius: 10px;
 }
+
+@media screen and (min-width: 768px) {
+  html,
+  body {
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    overflow-x: hidden;
+  }
+
+  .headline {
+    width: 1440px;
+    height: 870px;
+  }
+
+  .headline nav {
+    width: 1440px;
+    height: 72px;
+  }
```

Comment:
[OPTIONAL IMPORTANT TIP] I've noticed that you are using a fixed width for both your mobile and desktop version. Other reviewers should have warned you about this, as seen in the screenshot below: 

Mobile with a higher width:
![image](https://user-images.githubusercontent.com/42799579/193393380-6fef29d2-ae69-4978-846f-79819081b0ee.png)

Desktop with a higher width:
![image](https://user-images.githubusercontent.com/42799579/193393260-9a66d56c-d7f5-41b3-a88d-4915a67577ce.png)

Desktop with a lower width:
![image](https://user-images.githubusercontent.com/42799579/193393329-c3e8ee49-ad98-4a0a-9e16-d8b56817158d.png)

Note that our webpages should be responsive, this means that they should _adapt_ to any devices that are trying to display them. For this reason, it is a good practice to use percentages to set your page width, making it adapt depending on the screen size. If you give a fixed width to a webpage, devices with a lower/higher width will have problems displaying it.
Your mobile version shouldn't be displayed with a short width, it should occupy the whole screen but be displayed better on smaller ones. You shouldn't worry about how it looks on a much larger screen because media queries will do the job (desktop version, for example).
Following the same logic, your desktop version shouldn't be displayed on a large width, it should also occupy the whole screen and adjust itself. Again, you shouldn't worry about how it looks on smaller ones, the mobile query will do the job. 😃
Please try to stick with this principle: responsiveness. You shouldn't make your project look perfect on just one screen, but make it work great on the majority of the screens. I know that this will give you some more work by now, but I'm sure that this will help you a lot in the future. Give this [link](https://www.w3schools.com/html/html_responsive.asp) and this [one] (https://www.youtube.com/watch?v=lAOkx2yZESY) a look if you are interested. Here is also a list of very good websites [examples]https://www.invisionapp.com/inside-design/examples-responsive-web-design/) of responsive design. Good luck on your web dev journey, hope this helped you in some way and hope you the best! 😉